sdk/healthdataaiservices/Azure.ResourceManager.HealthDataAIServices/README.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure HealthDataAIServices management client library for .NET
+
+> Azure Health Deidentification is a managed service that enables users to tag, redact, or surrogate health data.
```

Comment:
is this the right name of our service?
I saw the namespace is `Azure.ResourceManager.HealthDataAIServices`, but here we are calling ourselves as `Azure Health Deidentification`

Reply:
Our naming convention has been in flux for a while. We are going to solidify and sweep through and update each to be consistent next release.